api_compatibility_policy.md

```diff
@@ -90,26 +90,25 @@ For more information on support windows, see the [deprecations table](./docs/dep
 
 ## Feature Gates
 
-CRD API versions gate the overall stability of the CRD and its default behaviors. Within a particular CRD version, certain opt-in features may be at a lower stability level as described in [TEP-33](https://github.com/tektoncd/community/blob/main/teps/0033-tekton-feature-gates.md). These fields may be disabled by default and can be enabled by setting the right `enable-api-fields` feature-flag as described in TEP-33:
+Stability levels of feature gates are independent from CRD apiVersions. The current group api-driven feature flag `enable-api-fields` gates the overall stability levels of features in the CRDs and their default behaviors. These fields may be disabled by default and can be enabled by setting the `enable-api-fields` feature-flag value:
```

Comment:
Hmm for this part I was trying to call out api driven fields were guarded behind `enable-api-fields`. I think this would introduce ambiguity on whether other behavioural flags would also need to set `enable-api-fields` to `alpha` to get turned on? I wanted also to add some doc for behavioural flags after we introduce per feature flag but not to conflate them here. Please lemme know if that makes sense.

Reply:
Thanks, updated!